docs/certificates.md

```diff
@@ -0,0 +1,20 @@
+# Certificates
+
+The Cross-Media Measurement API uses X.509 certificates for digital signatures.
+The certificates returned in API resources are never root certificates, and
+should be verified using a certificate that has been exchanged outside of the
+API.
+
+## Certificate Preference
+
+Some resources have a field that refers to a *preferred* certificate resource.
+The following criteria, in order of most to least significant, determine
+certificate preference within a collection:
+
+1.  Not revoked.
+1.  Within its validity period.
+1.  Has a later end to its validity period.
```

Comment:
Is there a best-practice reason why we are putting later end certificates with higher precedence above later start certificates? A later-start date implies a more recently created certificate which should have higher precedence.

Reply:
Good question. While it's true that a later start date implies a more recent certificate, what the system is concerned with is reducing the likelihood that a certificate expires prior to the completion of all operations that are using it. Picking the certificate with the longest remaining time to expiration better accomplishes this.

In practice, we expect that certificates with later end dates will also have later start dates, so the precedence between those two items would be moot.